lib/xmldoc.js

```diff
@@ -135,7 +135,10 @@ XmlElement.prototype.toStringWithIndent = function(indent, options) {
     if (Object.prototype.hasOwnProperty.call(this.attr, name))
         s += " " + name + '="' + this.attr[name] + '"';
 
-  var finalVal = this.val.trim().replace(/</g, "&lt;").replace(/>/g, "&gt;");
+  var finalVal = this.val.trim().replace(/</g, "&lt;").replace(/>/g, "&gt;")
+      .replace(/&/g, '&amp;')
+      .replace(/"/g, '&quot;')
```

Comment:
Good catch on the ampersand - that does need to be escaped. But I don't believe quotes need to be escaped for XML text nodes. Try it here: http://www.w3schools.com/xml/xml_validator.asp